scripts/move_canvas_files_to_amazon_s3.js

```diff
@@ -186,42 +186,94 @@ var storeAssetInAmazonS3 = function(asset, stream, filePath, callback) {
 };
 
 /**
- * Move file asset to Amazon S3.
+ * @param  {Course}       whiteboard       Whiteboard with image URL
+ * @param  {Course}       stream           The content being moved (input stream)
+ * @param  {Course}       filePath         Where to write content on local disk
```

Comment:
@johncrossman Minor - `{Course}` not courses

Reply:
Thanks, @paulkerschen  Fixed.